src/dev/build/args.ts

```diff
@@ -40,6 +40,7 @@ export function readCliArgs(argv: string[]) {
       'silent',
       'debug',
       'help',
+      'use-snapshot-registry',
```

Comment:
It might be useful to make this more specific e.g. `use-epr-snapshot-registry`.  The cloud dependencies task for example will toggle between snapshots for metricbeat/filebeat if the `--release` flag is used, and could cause some confusion.

Reply:
Good thought - I've updated this to `use-snapshot-epr` for clarity.